src/config/__jest__/Config.test.ts

```diff
@@ -0,0 +1,99 @@
+import fs from 'fs';
+import Config from '../Config';
+import lodash from 'lodash';
+
+jest.mock('fs');
+describe('Default variables', () => {
+  let testedConfig = new Config();
```

Comment:
`const` for variables that are not re-assigned.

I usually test with `it`, e.g. `it('sets default clientId and token to empty strings')`

Could you also add some new lines here and there, e.g between test cases?